SConstruct

```diff
@@ -932,20 +961,16 @@ if env['system_eigen'] in ('n', 'default'):
                          'Try manually checking out the submodule with:\n\n'
                          '    git submodule update --init --recursive ext/eigen\n')
 
+if env['system_eigen']:
+    eigen_include = ['<Eigen/Core>', '<Eigen/src/Core/util/Macros.h>']
+else:
+    eigen_include = ['"../ext/eigen/Eigen/Core"', '"../ext/eigen/Eigen/src/Core/util/Macros.h"']
```

Comment:
`Eigen/Core` already has `#include "src/Core/util/Macros.h"`. Is there a reason to explicitly include this?